pom.xml

```diff
@@ -1,16 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.1.10</version>
+    <relativePath />
+    <!-- lookup parent from repository -->
+  </parent>
   <groupId>de.doubleslash</groupId>
   <artifactId>keeptime</artifactId>
-  <version>${project.version}</version>
-  <packaging>jar</packaging>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
version in build artifact and filename is not replaced like this (should use version passed from mvn package commandline)